scripts/bug_retriever.py

```diff
@@ -207,9 +207,55 @@ def retrieve_bugs(self, limit: int | None = None) -> None:
             len(missing_history_bug_ids),
         )
 
+        handel_missing_comments()
+
         zstd_compress(bugzilla.BUGS_DB)
 
 
+def handel_missing_comments(trail_number: int = 1, max_tries: int = 2) -> int:
```

Comment:
```suggestion
def handle_missing_comments(trial_number: int = 1, max_tries: int = 2) -> int:
```

Reply:
Fixed in 8639b1ce0a8e03f1ddc6c6dac8af122c7a9e6c0f